src/components/app/index.tsx

```diff
@@ -93,6 +99,18 @@ export default class App extends Component<Props, State> {
     const fileInput = event.target as HTMLInputElement;
     const sourceFile = fileInput.files && fileInput.files[0];
     if (!sourceFile) return;
+    await this.updateFile(sourceFile);
+  }
+
+  @bind
+  async onFileDrop(event: FileDropEvent) {
+    const sourceFile: File = event.file;
+    if (!sourceFile) return;
+    this.setState(initialState());
```

Comment:
Create an issue to remove this line and fix the issues it causes.

Reply:
Issue raised. #63